project.clj

```diff
@@ -4,5 +4,7 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :repositories [["choco.repos" "http://www.emn.fr/z-info/choco-repo/mvn/repository/"]]
-  :dependencies [[org.clojure/clojure "1.5.1"]
-                 [choco/choco-solver "3.1.0"]])
+  :dependencies [[org.clojure/clojure "1.7.0-beta1"]
+                 [choco/choco-solver "3.1.0"]
+                 [criterium "0.4.3"]]
```

Comment:
I'd rather not introduce a dependency on criterium just for the tests. Please either remove all the criterium-related code, or add this to a "test" profile so the main source code doesn't depend on it. The first solution would be easier, though.

Same goes for `refactor-nrepl`, you can easily put that in your `~/.lein/profiles.clj` instead of this `project.clj`.